diff --git a/index.js b/index.js
index 709991da99b8b5cd9c5f4d81774d874da087a20c..d29f2ec2ce61473e390c18e289216060a0025c19 100644
--- a/index.js
+++ b/index.js
@@ -12,7 +12,7 @@ function isParentFolder(relativeFilePath, context, rootDir) {
 }
 
 function isSameFolder(path) {
-  return path.startsWith("./");
+  return path.startsWith("./") || path === ".";
 }
 
 function getRelativePathDepth(path) {
